youboraplugin/src/main/java/com/kaltura/playkit/plugins/youbora/YouboraPlugin.java

```diff
@@ -149,14 +150,33 @@ protected void onUpdateMedia(PKMediaConfig mediaConfig) {
         }
     }
 
+    private void addKalturaInfoProperty(PKMediaConfig mediaConfig) {
+        if (npawPlugin != null &&
+                npawPlugin.getOptions() != null &&
+                npawPlugin.getOptions().getContentMetadata() != null &&
+                player != null &&
+                mediaConfig != null &&
+                mediaConfig.getMediaEntry() != null &&
+                mediaConfig.getMediaEntry().getMetadata() != null
+        ) {
+            Bundle kalturaInfoBundle = new Bundle();
+            kalturaInfoBundle.putString("entryId", mediaConfig.getMediaEntry().getMetadata().get("entryId"));
+            kalturaInfoBundle.putString("sessionId", player.getSessionId());
+            kalturaInfoBundle.putString("uiConfId", "");
```

Comment:
all of this data is optional, no need to add this as we don't have uiConfId on SDK at all ATM

Reply:
removed uiConfId